CONVERT_SERVICE/convert_service/convert_service/urls.py

```diff
@@ -13,9 +26,21 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from convert_service.views import index, upload, download
```

Comment:
Please review PEP8 importing rules here.

Reply:
Thank you, I will review the code.